<?xml version="1.0"?>
<launch>
    <arg name="map_file" default="$(find mcl)/mapdata/map.yaml" />

    <!-- map server -->
    <node pkg="map_server" type="map_server" name="global_map" args="$(arg map_file)" />

    <!-- mcl -->
    <node pkg="mcl" type="mcl_node" name="mcl_node" output="screen" >
        <param name="scan_topic_name" value="/scan" />
        <param name="map_topic_name" value="/map" />
        <param name="est_pose_topic_name" value="/est_pose" />
        <param name="est_poses_topic_name" value="/est_poses" />
        <param name="map_frame_id" value="map" />
        <param name="odom_frame_id" value="odom" />
        <param name="base_link_frame_id" value="base_link" />
        <rosparam command="load" file="$(find mcl)/config/mcl/mcl.yaml" />
    </node>

</launch>